src/Microsoft.AspNetCore.Blazor.Server/Circuits/MessagePackBufferStream.cs

```diff
@@ -8,28 +8,19 @@
 namespace Microsoft.AspNetCore.Blazor.Server.Circuits
 {
     /// <summary>
-    /// A write-only stream that outputs its data to an underlying expandable
-    /// buffer in the format for a MessagePack 'Bin32' block. Supports writing
-    /// into buffers up to 2GB in length.
+    /// Provides Stream APIs for writing to a MessagePack-supplied expandable buffer.
     /// </summary>
-    internal class MessagePackBinaryBlockStream : Stream
+    internal class MessagePackBufferStream : Stream
     {
-        // MessagePack Bin32 block
-        // https://github.com/msgpack/msgpack/blob/master/spec.md#bin-format-family
-        const int HeaderLength = 5;
```

Comment:
We no longer have to format the actual Bin32 block ourselves, because we're passing the `byte[]` to SignalR's `SendAsync` (instead of passing a `RenderBatch`), so SignalR now calls the appropriate MessagePack APIs to format the `byte[]` as a Bin32 block.

Hence we no longer need to write out the header manually, and I've removed all the code that did that.

Reply:
Reopened #1132